lib/index.js

```diff
@@ -53,19 +53,56 @@ class PercySeleniumClient {
     await this.uploadMissingResources(buildResponse, shaToResource);
   }
 
+  async getPageSource() {
+    const { driver } = this;
```

Comment:
💀 **BIKESHEDDING** 💀

I don't know, there is something really odd about destructuring this :D